### ISSUE TITLE
Screen reader support

### ISSUE THREAD

**Issue by merenyia** [notepad-plus-plus/notepad-plus-plus#11486]
Hi,
after upgrading Notepad++ to 8.3.3. (64 bit) The application is unusable with screen reader. Menu bar is accessible but the editor part is simply "looks" empty. I mean like it would be empty.
I use Jaws and tried Narrator but reader doesn't read anything.

### Steps to Reproduce the Issue

1. Open screen reader,
2. Open an existing and not empty txt file with Notepad++
3. On the editor area read content on usual way as it usual with the reader.
4. 4. The reader will read nothing.

### Expected Behavior

Screen reader should read content as it was before upgrade.

**Comment by xomx** [comment]
#11424
